### PR TITLE
Spring boot 2.4 upgrade

### PR DESCRIPTION
**Pull request by skamoen**

### CODE REVIEW

File: src/test/java/ch/wisv/areafiftylan/integration/AuthenticationIntegrationTest.java [hunk 1]
```diff
@@ -122,8 +122,8 @@ public void testRequestTokenInvalidRequest() {
             body(userDTO).contentType(ContentType.JSON).
             post("/login").
         then().
-            statusCode(HttpStatus.SC_INTERNAL_SERVER_ERROR).
-            body("message", containsString("Cant read request data"));
```
Comment (martijnjanssen) [inline src/test/java/ch/wisv/areafiftylan/integration/AuthenticationIntegrationTest.java:126]: Spring no longer gives an error here?

File: src/test/java/ch/wisv/areafiftylan/integration/UserRestIntegrationTest.java [hunk 2]
```diff
@@ -257,7 +257,7 @@ public void testCreateUserIsDisabled() {
             post("/login").
         then().
             statusCode(HttpStatus.SC_UNAUTHORIZED).
-            body("message", containsString("Unauthorized"));
```
Comment (martijnjanssen) [inline src/test/java/ch/wisv/areafiftylan/integration/UserRestIntegrationTest.java:260]: Or should the field change to this?

**Review by martijnjanssen — APPROVED** [review]

File: src/test/java/ch/wisv/areafiftylan/integration/AuthenticationIntegrationTest.java (hunk 1 above)
Comment (skamoen) [reply src/test/java/ch/wisv/areafiftylan/integration/AuthenticationIntegrationTest.java:126]: Seems the message field is empty:
```
{
    "timestamp": "2020-12-17T11:35:23Z",
    "status": 500,
    "error": "Internal Server Error",
    "message": "",
    "path": "/login"
}
```

File: src/test/java/ch/wisv/areafiftylan/integration/UserRestIntegrationTest.java [hunk 3]
```diff
@@ -257,7 +257,7 @@ public void testCreateUserIsDisabled() {
...
+            body("error", containsString("Unauthorized"));
```
Comment (martijnjanssen) [inline src/test/java/ch/wisv/areafiftylan/integration/UserRestIntegrationTest.java:260]: Or should the field change to this?

File: src/test/java/ch/wisv/areafiftylan/integration/AuthenticationIntegrationTest.java (hunk 1 above)
Comment (martijnjanssen) [reply src/test/java/ch/wisv/areafiftylan/integration/AuthenticationIntegrationTest.java:126]: Okay, fair enough :)

**Pull request merged**